src/pages/ProductsPage/index.jsx

```diff
@@ -1,6 +1,5 @@
 import React, { useState } from "react";
 import Header from "../../components/Header";
-import Layout from "../../components/Layout";
 import { FiltersProvider } from "../../context/FilterContext";
 import { ProductsProvider } from "../../context/ProductsContext";
```

Comment:
Suggestion: you can use absolute import here for better practice . and it will help in understanding the path of files and it will reduce your lines of code too

Reply:
Like exporting all the provider from a single file